### PR TITLE
Add redirect.

### PR DESCRIPTION
**Pull request by YKo20010**
Cornell SSO button click redirects.

Also, I tried changing my VSCode tab to 2 spaces instead of 4 in the settings, but it changed nothing. All the files are 4-space indented on my side. Idk how to fix my tabs.

### CODE REVIEW

**Review by ywsang — APPROVED** [review]
ship it

File: src/components/PollingApp/index.tsx [hunk 1]
```diff
@@ -44,13 +45,15 @@ import {
...
 import './styles.scss';
+import { cornellSSOUrl } from '../../utils/constants';
 
 export interface PollingAppState {
     isComposingGroup: boolean;
     isCreatingGroup: boolean;
     isComposingPoll: boolean;
     isStartingPoll: boolean;
     isLoading: boolean;
+    shouldRedirect: boolean,
```
Comment (danielVebman) [inline src/components/PollingApp/index.tsx:56]: You use a comma here but semicolons elsewhere. In any case, typescript does not require semicolons. ￼

**Review by danielVebman — APPROVED** [review]
Noice 🚀

**Comment by danielVebman** [comment]
Kill the semicolons though! (No?)￼

File: src/components/PollingApp/index.tsx (hunk 1 above)
Comment (YKo20010) [reply src/components/PollingApp/index.tsx]: Woops. I actually changed shouldRedirect to have a semicolon for consistency with the rest of the files.

**Pull request merged**